src/services/groupServices.js

```diff
@@ -7,12 +7,18 @@ export default {
     getGroup(groupId) {
         return apiClient.get(`/groups/${groupId}`);
     },
+    getGroupWithPermissions(groupId) {
+        return apiClient.get(`/groups/${groupId}?full`);
+    },
     createGroup(data) {
         return apiClient.post(`/groups/`, data);
     },
     updateGroup(groupId, data) {
         return apiClient.put(`/groups/${groupId}`, data);
     },
+    updateGroupWithPermissions(groupId) {
```

Comment:
You need to pass in data to this function, otherwise how can the backend know what changes to make to the group or permissions?